lib/registration.pm

```diff
@@ -249,8 +249,13 @@ sub yast_scc_registration {
     my $ret = wait_serial "yast-scc-done-\\d+-", $timeout;
     die "yast scc failed" unless (defined $ret && $ret =~ /yast-scc-done-0-/);
 
-    script_run 'zypper lr';
-    assert_screen 'scc-repos-listed';
+    if (get_var('ONLINE_MIGRATION')) {
```

Comment:
Yes, I understand what you concerned.

What I thought this PR is just a temporary workaround in order to get results of scc/smt testings for RC1 while avoiding to assert many useless repo list needles:
https://openqa.suse.de/tests/overview?distri=sle&version=12-SP2&build=2075&groupid=45

This output was only for ONLINE_MIGRATION because of this is temporary and currently I don't want to affect other non online migration tests.

Also I caculated the workload to apply repo validity check by using zypper_lr, probably I need time to packaging it and may add some codes as well as modify plenty of online migration testsuites.
If it is feasible, please merge it.
And then I will implement repo validity check to replace this temporary workaround.